krypto/src/commonMain/kotlin/com/soywiz/krypto/encoding/Base64.kt

```diff
@@ -1,42 +1,58 @@
 package com.soywiz.krypto.encoding
 
 object Base64 {
-    private val TABLE = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/="
-    private val DECODE = IntArray(0x100).apply {
+    private const val TABLE = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/="
+    private const val TABLE_SAFE = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_="
+
+    private val decoder = IntArray(0x100).apply {
```

Comment:
Even if not const, this is still immutable and I see it as a constant table. I would use UPPERCASE here, or at least would be consisent for the TABLE names. Though if you did it with a criteria I'm open to see your point

Reply:
Done